extensions/QNX/EGL_QNX_platform_screen.txt

```diff
@@ -0,0 +1,89 @@
+Name
+
+    QNX_platform_screen
+
+Name Strings
+
+    EGL_QNX_platform_screen
+
+Contributors
+
+    Mike Gorchak <mgorchak@blackberry.com>
+    Jeff Vigil <j.vigil@samsung.com>
+
+Contacts
+
+    Mike Gorchak <mgorchak@blackberry.com>
+
+Status
+
+    Complete
+
+Version
+
+    Version 1, 2023-06-13
+
+Number
+
+    EGL Extension #150
+
+Extension Type
+
+    EGL client extension
+
+Dependencies
+
+    Requires EGL_EXT_client_extensions to query its existence without
+    a display.
+
+    Requires EGL_EXT_platform_base.
+
+    This extension is written against the wording of version 7 of the
+    EGL_EXT_platform_base specification.
+
+Overview
+
+    This extension defines how to create EGL resources from native qnx
+    resources using the functions defined by EGL_EXT_platform_base.
+
+New Types
+
+    None
+
+New Procedures and Functions
+
+    None
+
+New Tokens
+
+    Accepted as the <platform> argument of eglGetPlatformDisplayEXT:
+
+        EGL_PLATFORM_SCREEN_QNX                0x3550
```

Comment:
@stonesthrow this wasn't actually defined in egl.xml so header generation failed. I just pushed a commit to main that fixes that and updates the headers - please verify it is as you desire but it is consistent with this assignment.

Reply:
I thought egl.h and eglext,h were generated from egl,xml? not hand edited.
OK. thank you for fix.
I will get #181 fixed right. It will be up in a few minutes.

Reply:
They are generated from egl.xml. egl.xml was missing the new enum EGL_PLATFORM_SCREEN_QNX - there was a <require> tag for it, but no actual definition of the value.